docs/source/references/config_files/project/base_yml.rst

```diff
@@ -258,6 +328,13 @@ this.
       port: 5672
       user: omniport_user
       password: omniport_password
+  emails:
+    emailBackend: 'django.core.mail.backends.smtp.EmailBackend'
+    emailHost: 'smtp.example.com'
+    emailUseTls: True
+    emailPort: 587
+    emailHostUser: 'img@iitr.ac.in'
+    emailHostPassword: 'img@password'
```

Comment:
Email user is not maintainers but a separate email id of the site. Generally, these emails are no-reply emails which are not used for daily life communication but the purpose of the notification only. no-reply@omniport.com will propagate a better idea in my opinion.